open_seq2seq/utils/utils.py

```diff
@@ -522,6 +524,8 @@ def get_base_config(args):
   if base_config is None:
     raise ValueError('base_config dictionary has to be '
                      'defined in the config file')
+  base_config['use_xla_jit'] = args.use_xla_jit
```

Comment:
can you please change it to 
```python
base_config['use_xla_jit'] = args.use_xla_jit or base_config.get('use_xla_jit', False)

This way if the user specified 'use_xla_jit" in config but not command line, it will still work with command line having an option to override it to true
```

Reply:
Sounds good to me. Revised.